docs/docs/pyproject.md

```diff
@@ -173,6 +173,17 @@ If a VCS is being used for a package, the exclude field will be seeded with the
 include = ["CHANGELOG.md"]
 ```
 
+You can also use inline tables for `include`:
```

Comment:
```suggestion
You can also specify the formats for which these patterns have to be included, as shown here.
```

Should we add a multi-format entry to the example?

Reply:
yes, good idea